source/SIL.AppBuilder.Portal.Frontend/src/redux-store/user-interface/actions/set-active-menu.ts

```diff
@@ -0,0 +1,13 @@
+import { namespace, State } from '../shared';
```

Comment:
I don't think you need `set-active-menu`, because the sidebar should use `react-router`'s `NavLink` for the built-in is-active functionality: https://reacttraining.com/react-router/web/api/NavLink/isactive-func

you'll just need to create route stubs for each of the links (empty route files)